templates/taxi/car_detail.html

```diff
@@ -3,20 +3,32 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <form action="{% url 'taxi:car-driver-manage' pk=car.id %}" method="post">
+		{% csrf_token %}
+		<div class="float-right">
+		{% if user in car.drivers.all %}
```

Comment:
The condition `{% if user in car.drivers.all %}` checks if the user is in the car's drivers list. Ensure that the `user` object is correctly compared with the driver objects in the list. This may require ensuring that the `user` object is the same type as the objects in `car.drivers.all`.